5-network/05-fetch-crossorigin/article.md

```diff
@@ -1,160 +1,174 @@
-# Fetch: Cross-Origin Requests
+# Fetch: Cross-Origin *request*s
+# Fetch: *request Cross-Origin*
 
-If we send a `fetch` request to another web-site, it will probably fail.
+Jika kita mengirim *request* *`fetch`* dari situs web lain, itu mungkin akan gagal.
 
-For instance, let's try fetching `http://example.com`:
+Misalnya, mari coba *fetch* dari `http://example.com`:
 
 ```js run async
 try {
   await fetch('http://example.com');
 } catch(err) {
-  alert(err); // Failed to fetch
+  alert(err); // Gagal untuk *fetch*
```

Comment:
Yang _Failed to fetch_ nya ini keluaran dari Javascripnya, mending gausah di translate tapi dikasih catatan tambahan jadi kaya:
Failed to fetch (gagal untuk *fetch* )